telecine/build.gradle

```diff
@@ -79,8 +79,8 @@ dependencies {
   apt 'com.jakewharton:butterknife-compiler:8.0.1'
   compile 'com.jakewharton.timber:timber:4.1.2'
   compile 'com.bugsnag:bugsnag-android:3.4.0'
-  compile 'com.squareup.dagger:dagger:1.2.5'
-  apt 'com.squareup.dagger:dagger-compiler:1.2.5'
+  compile 'com.google.dagger:dagger:2.2'
+  apt 'com.google.dagger:dagger-compiler:2.2'
```

Comment:
Thanks for the quick review. Let me update.
On Jun 14, 2016 7:10 AM, "Jake Wharton" <notifications@github.com> wrote:

> In telecine/build.gradle
> <https://github.com/JakeWharton/Telecine/pull/115#discussion_r66977034>:
>
> > @@ -79,8 +79,8 @@ dependencies {
> >    apt 'com.jakewharton:butterknife-compiler:8.0.1'
> >    compile 'com.jakewharton.timber:timber:4.1.2'
> >    compile 'com.bugsnag:bugsnag-android:3.4.0'
> > -  compile 'com.squareup.dagger:dagger:1.2.5'
> > -  apt 'com.squareup.dagger:dagger-compiler:1.2.5'
> > +  compile 'com.google.dagger:dagger:2.2'
> > +  apt 'com.google.dagger:dagger-compiler:2.2'
>
> 2.4 is latest
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JakeWharton/Telecine/pull/115/files/688b691721cbcb761fef90dbb8316eea24f1f9b3#r66977034>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABqMSBNROkXc5oC8GufXlRQcTr7HrxFYks5qLrZBgaJpZM4I0-8d>
> .
>